sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -620,6 +620,15 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
     .stringConf
     .createOptional
 
+  val TIMESTAMP_RULES_END_YEAR = conf("spark.rapids.timezone.transitionCache.maxYear")
+    .doc("Set the max year for timestamp processing for timezones with transitions " +
+      "such as Daylight Savings. For efficiency reasons, timestamp" +
+      " transitions are stored on the GPU. We store transitions up to some set year." +
+      " Adding more years will use more memory, every 100 years is roughly 1MB.")
+    .commonlyUsed()
```

Comment:
I'm not sure that this is commonly used. Also this needs to be set at startup so we usually mark it with `.startupOnly()`